src/Services/Device.vala

```diff
@@ -286,6 +286,9 @@ public class Power.Services.Device : Object {
             return _("Calculating…");
         }
 
+        if (percent == 100) {
```

Comment:
Shouldn't it also check if the battery is charging as well? I'd expect to see "100%" when fully charged but unplugged, and "Charged" when fully charged but connected to power.

Reply:
Yep. Let me make the change